docs/configuration.md

```diff
@@ -74,7 +74,7 @@ The following format is accepted:
     1p or 1pb (pebibytes = 1024 tebibytes)
 
 While numbers without units are generally interpreted as bytes, a few are interpreted as KiB or MiB.
-See documentation of individual configuration properties. Specifying units is desirable where 
+See documentation of individual configuration properties. Specifying units is desirable where
```

Comment:
BTW there are many changes for trimming spaces in this PR. I go over them and they look fine. It would be great to update the editor to only trim the spaces of the modified lines in the future.

Reply:
Will do.